client/src/components/App.js

```diff
@@ -1,36 +1,43 @@
 /**
  * Root component.
  */
-import React from 'react';
-import {BrowserRouter, Switch, Route} from 'react-router-dom';
+import React from "react";
+import { Toaster } from "react-hot-toast";
```

Comment:
I think you haven't pushed the updated `package.json` file. I got an error saying `Module not found: Can't resolve 'react-hot-toast' in....` when I tried running it. I tried doing `npm install` and it didn't fix it so what I did was `npm install react-hot-toast` and that added a new line in `package.json`. Would ideally be better to add this in before doing the merge.